IsogeomGenerator/vol.py

```diff
@@ -696,21 +598,21 @@ def _compare_surfs(self, v1, v2):
                     self.surf_curve[s2] = []
 
                 # get list of all coordinates in s2 (inverted)
-                verts2_inv = self._list_coords(s2, invert=True)
+                verts2_inv = self.__list_coords(s2, invert=True)
 
                 # compare vertices and gather sets for s1 and s2
                 # that are coincident
-                s1_match_eh, s1_match_coords, s2_match_eh, s2_match_coords, match_dict =\
-                    self._get_matches(verts1, verts2_inv)
+                s1_match_eh, s1_match_coords, s2_match_eh, s2_match_coords, \
+                    match_dict = self.__get_matches(verts1, verts2_inv)
 
                 if s1_match_eh != []:
                     # matches were found, so continue
 
```

Comment:
why a blank line ?

Reply:
see above comment- I am not making those changes in this PR